AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -463,6 +476,32 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
 
 
 #pragma mark -
+#pragma mark Orientation Change Handling
+
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
+{
+  _pendingRelayoutForAllRows = YES;
```

Comment:
Yes, you are right. Should check for any changes before setting this flag. Fixed in the new PR.